.github/workflows/pr.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     with:
       npmTag: rc
       buildScript: build
-      nodeVersion: 16
+      nodeVersion: 20
```

Comment:
Just a side note, I think at some point we can change this in `shared-config`; we can read the `node-version` file and just use it by default

Reply:
We can also use a .nvmrc file and point to it during setup-node with `node-version-file` option.